firmware/fw_base.S

```diff
@@ -119,7 +119,7 @@ _prev_arg1_override_done:
 	 */
 	beqz	a1, _fdt_reloc_done
 	/* Mask values in a3 and a4 */
-	li	a3, ~0xf
+	li	a3, ~0x7
```

Comment:
Thanks for catching.

How about (__SIZEOF_POINTER__ - 1) instead of ~0x7 ??

Reply:
Thanks, this suggestion is great.